docs/api/SharedStyle.md

```diff
@@ -109,9 +109,10 @@ You can unlink a Shared Style from the Library it comes from and make it a local
 SharedStyle.StyleType.Text
 ```
 
-Enumeration of the type of Shared Style.
+Enumeration of the type of Shared Style. Note that the `Unknown` shouldn't happen. It's a sign that the object is broken.
```

Comment:
Let's remove _"(`Layer` or `Text`)"_ [above](https://github.com/BohemianCoding/SketchAPI/pull/387/files#diff-e332d3cb55f690441912233ba05205caR23) and leave the details to `SharedStyle.StyleType`.

> The type of the Shared Style (`Layer` or `Text`).

Also, "shouldn't happen" isn't the best choice of words, I suggest:

> Enumeration of the type of Shared Style. `Unknown` indicates the object is broken and Sketch can't determine the style type.